config_example.toml

```diff
@@ -5,9 +5,11 @@ default_prefix = "-dev" # all guilds will use this prefix unless changed
 shard_count = 1 # amount of shards jim will create
 
 [database]
-user = "" # put your postgres username here
+user = "postgres" # put your postgres username here
 pass = "" # put your postgres password here
-jdbc_url = "jdbc:postgresql://(host_name):(port)/(database_name)"
+host = "database"
+port = 5432
+database = "postgres"
```

Comment:
We can comment here that these are defaults for docker deployment of Jim (maybe even mention it more explicitly in the README).